core/coreapi/interface/tests/block.go

```diff
@@ -103,16 +116,17 @@ func TestBlockGet(t *testing.T) {
 	}
 }
 
-func TestBlockRm(t *testing.T) {
-	ctx := context.Background()
-	_, api, err := makeAPI(ctx)
+func (tp *provider) TestBlockRm(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	api, err := tp.makeAPI(ctx)
 	if err != nil {
 		t.Error(err)
 	}
 
 	res, err := api.Block().Put(ctx, strings.NewReader(`Hello`))
 	if err != nil {
-		t.Error(err)
+		t.Fatal(err)
```

Comment:
What's the pattern for the replacement of Error with Fatal? Are we testing the first use of the CoreAPI generally?

Reply:
I think I understand - it's an existing inconsistency and you didn't want to change them wholesale.